java/java-guestbook/backend/src/main/java/cloudcode/guestbook/backend/BackendApplication.java

```diff
@@ -0,0 +1,20 @@
+package cloudcode.guestbook.backend;
```

Comment:
license and copyrights should be everywhere.  (Assume I've mentioned it for every file that isn't machine generated.

Reply:
As @simonz130 mentioned, we're releasing these as templates using the 0BSD license